volatility3/framework/plugins/linux/lsof.py

```diff
@@ -21,6 +21,8 @@ class Lsof(plugins.PluginInterface):
 
     _required_framework_version = (2, 0, 0)
 
+    _version = (1, 1, 0)
```

Comment:
This can start at (1, 0, 0), without an `_version` defined, I believe it defaults to (0, 0, 0) and is assumed not to contain any external methods that can be called.  It doesn't have to though, (1, 1, 0) is fine too.  5;)